circleci/images/exttester/files/sbin/install-testdeps

```diff
@@ -70,6 +73,12 @@ chmod o+w -R /usr/
 # install gosu so that we can run tests from root with circleci user
 apt-get install -y gosu
 
+# some tests rely on da_DK locale being available
```

Comment:
Should we say something like `some tests in some repos rely on da_DK locale being available` so that people are more cautious about removing this line?